README.md

```diff
@@ -1,70 +1,174 @@
-# Getting Started with Create React App
+<div align="center">
```

Comment:
- [ ] Your test is still failing. As you can see from the screenshot below and as highlighted by the previous reviewer, the failing test is in the `src/App.test.js` file. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/54848363/206153349-40a40b48-0d93-4cb0-8b7c-ef9611f06a4a.png)